docs/manual/generators.md

```diff
@@ -173,48 +173,65 @@ This mechanism can be used to select arbitrary sections of decay chains!
 
 To generate double beta decay physics we interface with the _bxdecay0_ package.
 This requires _remage_ to be build with _bxdecay0_ support see {ref}`install`.
-The only documentation for this extension is available
+The main documentation for this extension is available
 [here](https://github.com/BxCppDev/bxdecay0).
 
 The macro [command](project:../rmg-commands.md#rmggeneratorselect):
 
 `/RMG/Generator/Select BxDecay0`
 
-can be used to select the Decay0 generators.
+can be used to select the Decay0 generator.
 
-We can then use the _bxdecay0_ double beta decay generator macro commands to
-generate double beta decay physics. The macro command to generate double beta
-decay physics is:
+_remage_ adds two macro commands to control the generator, which should cover
+all required use cases. While the original _bxdecay0_ double beta decay
+generator macro commands are still available to the user, the _remage_ commands
+offer a simpler and more intuitive alternative.
 
-```
-/bxdecay0/generator/dbd ISOTOPE SEED MODE LEVEL
-```
-
-Where:
-
-- `ISOTOPE` (string): is the decaying nuclei,
-- `SEED` (int): is a random seed,
-- `MODE` (int): is an index the double beta decay mode labelling the decay type,
-  a table of implemented decay modes can be found in
-  [link](https://github.com/BxCppDev/bxdecay0/tree/develop?tab=readme-ov-file#list-of-supported-double-beta-decay-modes).
-- `LEVEL` (int): is a index of the energy level of the daughter nuclei, the list
-  of available levels are
-  [documented here](https://github.com/BxCppDev/bxdecay0/tree/develop?tab=readme-ov-file#list-of-daughter-nucleus-excited-states-in-double-beta-decay).
+The macro command to generate double beta decay physics
+<project:../rmg-commands.md#rmggeneratorbxdecay0dbd> expects the isotope name,
+the name of the process you want to simulate (like 2vbb or 0vbb) and an optional
+energy level for the daughter nucleus. All available isotopes and processes are
+listed [here](project:../rmg-commands.md#rmggeneratorbxdecay0dbd). The energy
+level defaults to zero and can therefore be omitted in most cases.
 
 For example we can generate two-neutrino double beta decay to the $0^+$ ground
 state of $^{76}$Ge with:
 
 ```
 /RMG/Generator/Select BxDecay0
-/bxdecay0/generator/dbd Ge76 1234 4 0
+/RMG/Generator/BxDecay0/dbd Ge76 2vbb
 ```
 
+The second macro command is used to generate background physics with _bxdecay0_.
```

Comment:
"background" is a nebulous name, why not explicitly say "radioactive decay" or so?